app/registries/foreman/access_permissions.rb

```diff
@@ -622,15 +622,6 @@
       :"api/v2/ssh_keys" => [:destroy]
   end
 
-  permission_set.security_block :personal_access_tokens do |map|
-    map.permission :view_personal_access_tokens,
-      :"api/v2/personal_access_tokens" => [:index, :show]
```

Comment:
These actions will need to be mapped somewhere (probably `view_user`) instead of removing authorization from PAT